services/core-api/tests/reports/resource/test_mine_reports_resource.py

```diff
@@ -66,7 +66,7 @@ def test_get_code_required_reports_for_mine(test_client, db_session, auth_header
     for report in get_data['records']:
         received_date = datetime.strptime(report['received_date'], '%Y-%m-%d')
 
-        assert (start_date <= received_date.date())
+        assert (start_date.date() <= received_date.date())
```

Comment:
this has been failing intermittently lately... Pretty strange that it's only sometimes just trying this out to see if it fixes it